.travis.yml

```diff
@@ -9,6 +9,4 @@ gemfile:
 - gemfiles/Gemfile.rubocop-next
 script:
   - bundle exec rspec spec
-  - bundle show rubocop
-  - bundle show rubocop-shopify
```

Comment:
👍 these don't seem particularly useful. I'm guessing they were added previously to help debug some kind of issue.

Reply:
Yes, I forgot to remove them in the last PR 🤦